Source/EasyNetQ.Tests/Mocking/MockBuilder.cs

```diff
@@ -1,123 +1,124 @@
 using EasyNetQ.ConnectionString;
 using EasyNetQ.Consumer;
-using EasyNetQ.DI;
-using EasyNetQ.LightInject;
 using EasyNetQ.Producer;
+using Microsoft.Extensions.DependencyInjection;
 using RabbitMQ.Client;
 
-namespace EasyNetQ.Tests.Mocking;
-
-public class MockBuilder : IDisposable
+namespace EasyNetQ.Tests.Mocking
```

Comment:
Please roll such changes back: there is no need to use block-scoped namespaces instead of file based.

Reply:
Done